### PR TITLE
Some random fixes

### PR DESCRIPTION
**Pull request by njtierney**
From our meeting :)

### CODE REVIEW

File: _targets.R
Comment (chitrams) [inline _targets.R:1]: What does it mean by set up a workspace when our code errors?

`tar_option_set(workspace_on_error = TRUE)`

Comment (njtierney) [reply _targets.R:1]: This facilitates exploring errors by saving a special workspace file - this page explains it more details: https://books.ropensci.org/targets/debugging.html#workspaces

Essentially this allows you to, after encountering an error, type:

```r
tar_workspace(target_name_that_errored)
```

And then it loads all of the targets that are required for that target to run into the environment, so you can more easily debug it. We can chat more about this tomorrow if you'd like :)

Comment (chitrams) [reply _targets.R:1]: Ooh thanks for pointing me to that page! That was really helpful :) Will experiment with it today.

**Comment by njtierney** [comment]
I think this has all been resolved separately - we can close this if you like? :)

**Comment by chitrams** [comment]
> I think this has all been resolved separately - we can close this if you like? :)

Yep :)

**Pull request closed without merging**